blueprints/ember-cli-fastboot/index.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env node */
+const recast = require('recast');
+const { readFileSync, writeFileSync } = require('fs');
+
+module.exports = {
+  description: '',
+  normalizeEntityName() {
+    // no-op
+  },
+
+  afterInstall() {
```

Comment:
In order to get the diffing, we'd need to do this _before_ `afterInstall`. There are a few ways, but they all kinda suck.

I think the simplest / best thing would be to dynamically generate the file in a temp directory before calling `super` in `install` and using that generated directory as our `filesPath`.

Reply:
Shared a small snippet in discord to try to explain better what I meant:

```js
filesPath() {
  return this._filesPath;
},

install() {
  // make a temp dir, probably with tmp
  // do your recasty stuff into that directory
  this._filesPath = tmp.dirSync();
  
  this._super.install.apply(this, arguments;
}
```